### DISCUSSION TITLE
dirac-dms-get-file may stuck due to issue with xrd client

### DISCUSSION THREAD

**Discussion by IgorPelevanyuk** [DIRACGrid/DIRAC#6560] [General]
Hello,

Recently we faced with an issue on downloading files from EOS over xrootd in our installation. Basically, a transfer gets stucked. This happens not every time and depends on the size of file. Bigger file - bigger probability of a stuck. I noticed that with standard xrdcp command problem preserves. But if I use xrd.wantprot=gsi,unix parameter instead of xrd.gsiusrpxy, problem is gone. I wrote an issue in xrootd github and received answer from developers(https://github.com/xrootd/xrootd/issues/1813). Probably this issue will be fixed sometime soon.  And I am interested, is there a way to update xrdcp client in DIRAC version 7.3? So, our users will just need to reinstall DIRAC client, and our jobs will start automatically to use new version of xrd.

**Comment by chaen** [comment]
Hi @IgorPelevanyuk 

We could get a more recent version of xroot if and only if you use the python 3 version of DIRAC 7.3 (so based on DIRACOS2)
